README.md

```diff
@@ -3,8 +3,19 @@ Joy-Con input driver for Linux
 
 The current main code, written in Go, lives in the `prog4/jcdriver` folder, as it was the fourth attempt at making this program.
 
-## Compiling
+## Basic Instructions (Linux/OSX)
 
+### Compiling
+#### OSX
+download and install the latest go binary from [golang.org](https://golang.org/dl/)
+```
+brew install git
```

Comment:
`xcode-select --install` ?

Reply:
Yeah that'd probably be better :D

Reply:
I added the `xcode-select --install` line, but left in the `brew install git` line as it is generally best practice to install brew via git in OSX. (better version control etc)